arch/arm/soc/cypress/Kconfig

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (c) 2018, Cypress
```

Comment:
I don't know what should it be here according to Linaro rules.

Reply:
> I don't know what should it be here according to Linaro rules.

Ok, as you submit this PR from @cypress.com emails, I guess copyright should be Cypress' too. Otherwise, deferring to @galak .